### DISCUSSION TITLE
How to edit background blur of mobile menu ?

### DISCUSSION THREAD

**Discussion by samyakb-mtechzilla** [fuma-nama/fumadocs#774] [Q&A] [answered]
I would like to disable background blur for mobile menu. After looking through codebase, I found this which had class `max-md:bg-fd-background/80` but I would like to override the class to customize. It does provide className through aside props, but I guess its not exposed to be customized.

https://github.com/fuma-nama/fumadocs/blob/74ba3a97398acde28754412d85f353e27c1e70fa/packages/ui/src/components/layout/sidebar.tsx#L111-L120

Is there any way to override the aside values ?

**Comment by fuma-nama** [comment, marked as answer]
I will recommend CSS selectors, the sidebar has an id to select from CSS. You can do further styling there.

**Reply by samyakb-mtechzilla** [reply]
Thank you, that worked.
